version

```diff
@@ -0,0 +1 @@
+0.1
```

Comment:
How is that file used?

Reply:
It's used by one of the pipeline steps and then it's passed to application versioning and GitHub tags. 
Every rebuild will add +1:
0.1.1
0.1.2 etc.